util/rewrite_all_hw.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Since the script is removed at the end, I thought it was a bit too much to split it into three. I can still do it if you think it's really necessary though?

Reply:
I meant to only have the script in the commit message so it's never added or removed as part of repository. This makes the script and what it produces atomic in git history, and cause less confusion when reviewing by commits.

Add and then remove is also fine to me, I don't have too strong of an opinion about this.

Reply:
Ah I see, that makes a lot of sense. I will do that.

Reply:
done